Assets/SEE/Tools/FaceCam/FaceCam.cs

```diff
@@ -671,6 +735,8 @@ private void FaceCamOnFrontToggleServerRpc(bool networkFaceCamOnFront)
         [ClientRpc]
         private void FaceCamOnFrontToggleClientRpc(bool networkFaceCamOnFront)
         {
+            Debug.Log($"[RPC] Client {NetworkManager.Singleton.LocalClientId} received FaceCamOnFrontToggleClientRpc from server with networkFaceCamOnFront={networkFaceCamOnFront}\n");
```

Comment:
Just a thought: There's a lot of these log outputs which are useful for debugging but aren't really needed by default. Maybe we can gate these behind a debug flag? Or are these just temporary until the FaceCam issues are resolved?

Reply:
These will be removed when the issue is resolved.

Reply:
I added #ifdefs.